src/Microsoft.Health.Dicom.Core.UnitTests/Features/Update/UpdateRequestValidatorTests.cs

```diff
@@ -54,9 +54,11 @@ public void GivenAValidDataset_WhenValidated_ThenItShouldSucceed(DicomDataset da
 
     [Theory]
     [MemberData(nameof(GetInvalidDicomDataset))]
-    public void GivenAnInvalidDataset_WhenValidated_ThenExceptionShouldBeThrown(DicomDataset dataset)
+    public void GivenAnInvalidDataset_WhenValidated_ThenSucceedsWithErrorsInFailedAttributesSequence(DicomDataset dataset)
     {
-        Assert.Throws<BadRequestException>(() => UpdateRequestValidator.ValidateDicomDataset(dataset));
+        DicomDataset failedSop = UpdateRequestValidator.ValidateDicomDataset(dataset);
+        DicomSequence failedAttributeSequence = failedSop.GetSequence(DicomTag.FailedAttributesSequence);
+        Assert.Single(failedAttributeSequence);
```

Comment:
Could we validate the message too?

Reply:
Good point. I added the check for verifying error comment too.